antenna/ext/fs/crashstorage.py

```diff
@@ -151,3 +153,39 @@ def save_crash(self, crash_report):
 
         # Save raw crash
         self.save_raw_crash(crash_id, raw_crash)
+
+    def load_crash(self, crash_id):
+        """Load crash data."""
+        raw_crash = {}
+        dumps = {}
+        dump_names_fn = self._get_dump_names_path(crash_id)
+
+        try:
+            with open(dump_names_fn, "rb") as fp:
+                dump_names = json.load(fp)
+        except (IOError, OSError):
```

Comment:
Holy moly! `IOError` isn't a thing anymore. I was thinking we should keep using `OSError` because of permission issues and other possibilities, but you're right--`FileNotFoundError` is the right thing to do here and everything else should bubble up unhandled because it suggests something is super wrong.